### PR TITLE
initialize i18n zh-cn

### PR DESCRIPTION
**Pull request by WanderWang**

### CODE REVIEW

**Comment by octogonz** [comment]
@WanderWang Thanks for making this PR! Very cool!

File: websites/rushstack.io/docusaurus.config.js [hunk 1]
```diff
@@ -86,6 +90,10 @@ const config = {
           src: 'images/rushstack.svg'
         },
         items: [
+          {
+            type: 'localeDropdown',
+            position: 'left',
```
Comment (octogonz) [inline websites/rushstack.io/docusaurus.config.js:95]: Does the dropdown control work for you?

I tried this:
1. Navigate to `http://localhost:3000/pages/heft/overview/`
2. Choose Zhongwen from the dropdown
3. It navigates to `http://localhost:3000/zh-cn/pages/heft/overview/`

However the dropdown still says "English" and the page is a 404:

image

Comment (octogonz) [reply websites/rushstack.io/docusaurus.config.js:95]: Maybe I am doing something wrong?

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.NotFound.p1": {
+    "message": "我们没有找到您正在寻找的页面。",
+    "description": "The first paragraph of the 404 page"
+  },
+  "theme.NotFound.p2": {
+    "message": "请联系本站点的拥有者并附上原始链接以便让他们知道他们的链接失效了。",
+    "description": "The 2nd paragraph of the 404 page"
+  },
+  "theme.AnnouncementBar.closeButtonAriaLabel": {
+    "message": "Close",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:23]:
```suggestion
    "message": "关闭",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    "message": "博客列表页导航",
+    "description": "The ARIA label for the blog pagination"
+  },
+  "theme.blog.paginator.newerEntries": {
+    "message": "新的内容",
+    "description": "The label used to navigate to the newer blog posts page (previous page)"
+  },
+  "theme.blog.paginator.olderEntries": {
+    "message": "旧的内容",
+    "description": "The label used to navigate to the older blog posts page (next page)"
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:48]: not sure agian `后一页`?

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  },
+  "theme.blog.post.readingTime.plurals": {
+    "message": "One min read|{readingTime} min read",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:51]:
```suggestion
    "message": "预计阅读时间 {readingTime} 分钟",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.archive.title": {
+    "message": "存档",
+    "description": "The page & hero title of the blog archive page"
+  },
+  "theme.blog.archive.description": {
+    "message": "存档",
+    "description": "The page & hero description of the blog archive page"
+  },
+  "theme.blog.paginator.navAriaLabel": {
+    "message": "博客列表页导航",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:39]:
```suggestion
    "message": "博客分页",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.BackToTopButton.buttonAriaLabel": {
+    "message": "回到顶部",
+    "description": "The ARIA label for the back to top button"
+  },
+  "theme.blog.archive.title": {
+    "message": "存档",
+    "description": "The page & hero title of the blog archive page"
+  },
+  "theme.blog.archive.description": {
+    "message": "存档",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:35]: I am not sure the description should be same as title. Will the page render duplicate `存档`?

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.post.readMore": {
+    "message": "阅读更多",
+    "description": "The label used in blog post item excerpts to link to full blog posts"
+  },
+  "theme.blog.post.paginator.navAriaLabel": {
+    "message": "Blog post page navigation",
+    "description": "The ARIA label for the blog posts pagination"
+  },
+  "theme.blog.post.paginator.newerPost": {
+    "message": "Newer Post",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:63]:
```suggestion
    "message": "前一篇",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.post.readingTime.plurals": {
+    "message": "One min read|{readingTime} min read",
+    "description": "Pluralized label for \"{readingTime} min read\". Use as much plural forms (separated by \"|\") as your language support (see https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/language_plural_rules.html)"
+  },
+  "theme.blog.post.readMore": {
+    "message": "阅读更多",
+    "description": "The label used in blog post item excerpts to link to full blog posts"
+  },
+  "theme.blog.post.paginator.navAriaLabel": {
+    "message": "Blog post page navigation",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:59]:
```suggestion
    "message": "博客导航",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.paginator.navAriaLabel": {
+    "message": "博客列表页导航",
+    "description": "The ARIA label for the blog pagination"
+  },
+  "theme.blog.paginator.newerEntries": {
+    "message": "新的内容",
+    "description": "The label used to navigate to the newer blog posts page (previous page)"
+  },
+  "theme.blog.paginator.olderEntries": {
+    "message": "旧的内容",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:47]:
```suggestion
    "message": "下一页",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.archive.description": {
+    "message": "存档",
+    "description": "The page & hero description of the blog archive page"
+  },
+  "theme.blog.paginator.navAriaLabel": {
+    "message": "博客列表页导航",
+    "description": "The ARIA label for the blog pagination"
+  },
+  "theme.blog.paginator.newerEntries": {
+    "message": "新的内容",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:43]:
```suggestion
    "message": "上一页",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.AnnouncementBar.closeButtonAriaLabel": {
+    "message": "Close",
+    "description": "The ARIA label for close button of announcement bar"
+  },
+  "theme.BackToTopButton.buttonAriaLabel": {
+    "message": "回到顶部",
+    "description": "The ARIA label for the back to top button"
+  },
+  "theme.blog.archive.title": {
+    "message": "存档",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:31]:
```suggestion
    "message": "归档",
```

File: websites/rushstack.io/i18n/zh-cn/code.json [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+  "theme.blog.post.paginator.navAriaLabel": {
+    "message": "Blog post page navigation",
+    "description": "The ARIA label for the blog posts pagination"
+  },
+  "theme.blog.post.paginator.newerPost": {
+    "message": "Newer Post",
+    "description": "The blog post button label to navigate to the newer/previous post"
+  },
+  "theme.blog.post.paginator.olderPost": {
+    "message": "Older Post",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/code.json:67]:
```suggestion
    "message": "后一篇",
```

File: websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current.json [hunk 13]
```diff
@@ -0,0 +1,1394 @@
...
+  "sidebar.docsSidebar.category.Contributing": {
+    "message": "贡献",
+    "description": "The label for category Contributing in sidebar docsSidebar"
+  },
+  "sidebar.docsSidebar.category.Help": {
+    "message": "帮助",
+    "description": "The label for category Help in sidebar docsSidebar"
+  },
+  "sidebar.apiSidebar.category.API Reference": {
+    "message": "API 参考",
```
Comment (chengcyber) [inline websites/rushstack.io/i18n/zh-cn/docusaurus-plugin-content-docs/current.json:35]:
```suggestion
    "message": "API 参考手册",
```

**Review by octogonz — APPROVED** [review]

**Comment by octogonz** [comment]
This is now merged+deployed.  @WanderWang thank you again for getting this started! :-)

**Pull request merged**